src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -373,6 +373,13 @@ void GraphOptimizer::FuseConvolutionMatMulDeconvAndBias(Graph& graph) {
         if (biasNode->getType() != Type::Input || !biasNode->isConstant() || biasNode->getChildEdges().size() != 1) {
             return false;
         }
+// ACL supports i32 bias only for int inputs
```

Comment:
the phrase is a bit confusing:
'Quantized acl convolution requires i32 bias'
Also, it should not be a problem to enable it.
We just need to specify i32 bias in precision mapping configuration.

Reply:
is i32 always required?
If so, instead of:
{{_u8, _u8 | _i8, _i32 | _dynamic, _u8},           {bypass(), bypass(), bypass(), bypass()}},
It should be something like:
{{_u8, _u8 | _i8, _any, _u8},           {bypass(), bypass(), just<i32>(), bypass()}},